src/stan_math_backend/Lower_functions.ml

```diff
@@ -409,7 +412,7 @@ let lower_standalone_fun_def namespace_fun
   let internal_fname = namespace_fun ^ "::" ^ fdname in
   let template =
     match fdsuffix with
-    | FnLpdf _ | FnTarget -> [TypeLiteral "false"]
+    | FnLpdf _ | FnTarget | FnJacobian -> [TypeLiteral "false"]
```

Comment:
Do stand-alone functions not need to compute the jacobian? It's surprising that `FnTarget` and `FnJacobian` both have `false` here because for `FnTarget` it means **include** normalization constants but for `FnJacobian` it means **exclude** jacobians.